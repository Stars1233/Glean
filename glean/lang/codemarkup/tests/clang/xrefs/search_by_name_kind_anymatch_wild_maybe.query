query: search.code.SearchByNameKindAndLanguage { name = _, language = Cpp, kinds = { just = _ } }
perf: true
transform: [gensort, []]
