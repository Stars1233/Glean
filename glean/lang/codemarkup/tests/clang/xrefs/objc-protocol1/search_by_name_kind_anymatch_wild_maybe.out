[
  "@generated",
  {
    "key": {
      "name": "call",
      "language": 0,
      "kinds": 13,
      "entity": {
        "cxx": {
          "defn": {
            "function_": {
              "key": {
                "declaration": {
                  "key": {
                    "name": {
                      "key": {
                        "name": { "key": { "name": { "key": "call" } } },
                        "scope": { "global_": { } }
                      }
                    },
                    "signature": {
                      "key": {
                        "returns": { "key": "void" },
                        "parameters": [
                          { "name": { "key": "s" }, "type": { "key": "J *" } }
                        ]
                      }
                    },
                    "source": {
                      "file": { "key": "test.m" },
                      "lineBegin": 16,
                      "columnBegin": 1,
                      "lineEnd": 19,
                      "columnEnd": 1
                    }
                  }
                },
                "isInline": false
              }
            }
          }
        }
      }
    }
  }
]